docs/reference/api/rest/introduction.md

```diff
@@ -46,4 +49,177 @@ Chances are you're looking for how to interact with **Items**, the atomic conten
 </div>
 </div>
 
----
\ No newline at end of file
+---
+
+## Relational Data
+
+<div class="two-up">
+<div class="left">
+
+### Fetching
+
+By default, Directus only retrieves the "root level" of an item. Relational data can be nested by using
+[the `fields` parameter](/reference/api/query/#fields).
+
+You can use a wildcard (`*`) to fetch all items in a given level, and use `*.*` to fetch everything on the next related
+level as well.
+
+::: tip Performance & Size
+
+While the fields wildcard is very useful for debugging purposes, we recommend using specifically selected fields for
+production use. By only requesting the fields you really need, you can drastically speed up the request, and reduce the
+overall output size.
+
+:::
+
+### Creating / Updating / Deleting
+
+##### Many-to-One
+
+Many-to-One relationships are fairly straightforward to manage relationally. You can simply submit the changes you want
+as an object under the relational key in your collection. For example, if you wanted to create a new featured article on
+your page, you could submit:
+
+```json
+PATCH /items/pages/about
+
+{
+	"featured_article": {
+		"title": "This is my new article!"
+	}
+}
+```
+
+This will create a new record in the related collection, and save it's primary key in the `featured_article` field for
+this item. To update an existing item, simply provide the primary key with the updates, and Directus will treat it as an
+update instead of a creation:
+
+```json
+PATCH /items/pages/about
+
+{
+	"featured_article": {
+		"id": 15,
+		"title": "This is an updated title for my article!"
+	}
+}
+```
+
+Seeing that the Many-to-One relationship stores the foreign key on the field itself, removing the item can be done by
+nullifying the field:
+
+```json
+PATCH /items/pages/about
+
+{
+	"featured_article": null
+}
+```
+
+##### One-to-Many (/ Many-to-Many / Many-to-Any)
+
+One-to-Many—and therefore Many-to-Many and Many-to-Any—relationships can be updated in one of two ways:
```

Comment:
One-to-Many, and therefore Many-to-Many and Many-to-Any, relationships can be updated in one of two ways: